csrc/bindings/all_to_all_ops.cpp

```diff
@@ -136,7 +136,7 @@ void combine(
     bool doRecv
 ) {
   _CHECK_TENSOR(2, outTokens);
-  TORCH_CHECK(outTokens.scalar_type() == at::kBFloat16, "outTokens must be of type BFloat16");
+  TORCH_CHECK(outTokens.scalar_type() == at::kBFloat16 || outTokens.scalar_type() == at::kHalf, "outTokens must be of type BFloat16 or Float16");
```

Comment:
nit: the repository includes a `clang-format` config, could you format this statement accordingly?

We haven't set up CI yet, but we'd run:
```
find csrc -type f -regex '.*\.\(h\|cpp\|cuh\|cu\)' -exec clang-format --dry-run -Werror {} +
```